### ISSUE TITLE
Use of Loss funtion

### ISSUE THREAD

**Issue by NomiMalik0207** [oval-group/smooth-topk#6]
Hello great work man.
please tell me about have your function registered with official pytorch built-in functions? 
if yes than how can i use it?

**Comment by lberrada** [comment]
Hi, thanks for your interest. The function is not part of the official pytorch package, but you can clone this repository and import the functions from the files. See #2 for a minimal example about how to use the loss function.

**Comment by NomiMalik0207** [comment]
How you set the Noise level ? I mean where you are setting the value of p?

**Comment by lberrada** [comment]
See [this line](https://github.com/oval-group/smooth-topk/blob/master/src/data/main.py#L69)

**Comment by NomiMalik0207** [comment]
ok.. thanks for your help. Actually in my case I was using CrossEntropy for classification and got 70% on a small dataset. But when I apply yours it give only 50%  with default parameters? How can I improve accuracy as compared to crossentropy??

**Comment by lberrada** [comment]
If this is not a code issue, there is only so much help I can provide.

That said, you can check the following:
- sanity check: when k=1 and tau=1, you should get results close to cross-entropy
- select the same k for which you optimize and for which you evaluate the top-k accuracy
- cross-validate the value of the temperature parameter tau (usually on a grid of powers of 10, with values like 10^-3 up to 10)
